{"level":"error","message":"Uncaught Exception Uncaught Exception in Index.js","stack":"Error: Uncaught Exception in Index.js\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:22:7)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Unhandled Rejection Something Failed Miserably","stack":"Error: Something Failed Miserably\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:24:26)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Unhandled Rejection aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa","stack":"Error: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:24:26)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Unhandled Rejection aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa","stack":"Error: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:24:26)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Uncaught Exception express is not defined","stack":"ReferenceError: express is not defined\n    at module.exports (F:\\Node Js practise\\vidly\\startup\\routes.js:2:11)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:20:28)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Uncaught Exception genres is not defined","stack":"ReferenceError: genres is not defined\n    at module.exports (F:\\Node Js practise\\vidly\\startup\\routes.js:5:28)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:20:28)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"code":"MODULE_NOT_FOUND","level":"error","message":"Uncaught Exception Cannot find module './routes/customers'\nRequire stack:\n- F:\\Node Js practise\\vidly\\startup\\routes.js\n- F:\\Node Js practise\\vidly\\index.js","requireStack":["F:\\Node Js practise\\vidly\\startup\\routes.js","F:\\Node Js practise\\vidly\\index.js"],"stack":"Error: Cannot find module './routes/customers'\nRequire stack:\n- F:\\Node Js practise\\vidly\\startup\\routes.js\n- F:\\Node Js practise\\vidly\\index.js\n    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)\n    at Module._load (node:internal/modules/cjs/loader:920:27)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\startup\\routes.js:2:19)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)"}
{"code":"MODULE_NOT_FOUND","level":"error","message":"Uncaught Exception Cannot find module './routes/customers'\nRequire stack:\n- F:\\Node Js practise\\vidly\\startup\\routes.js\n- F:\\Node Js practise\\vidly\\index.js","requireStack":["F:\\Node Js practise\\vidly\\startup\\routes.js","F:\\Node Js practise\\vidly\\index.js"],"stack":"Error: Cannot find module './routes/customers'\nRequire stack:\n- F:\\Node Js practise\\vidly\\startup\\routes.js\n- F:\\Node Js practise\\vidly\\index.js\n    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)\n    at Module._load (node:internal/modules/cjs/loader:920:27)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\startup\\routes.js:2:19)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)"}
{"code":"MODULE_NOT_FOUND","level":"error","message":"Uncaught Exception Cannot find module './routes/customers'\nRequire stack:\n- F:\\Node Js practise\\vidly\\startup\\routes.js\n- F:\\Node Js practise\\vidly\\index.js","requireStack":["F:\\Node Js practise\\vidly\\startup\\routes.js","F:\\Node Js practise\\vidly\\index.js"],"stack":"Error: Cannot find module './routes/customers'\nRequire stack:\n- F:\\Node Js practise\\vidly\\startup\\routes.js\n- F:\\Node Js practise\\vidly\\index.js\n    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)\n    at Module._load (node:internal/modules/cjs/loader:920:27)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\startup\\routes.js:2:19)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)"}
{"level":"error","message":"Uncaught Exception dbConnectionLink is not defined","stack":"ReferenceError: dbConnectionLink is not defined\n    at module.exports (F:\\Node Js practise\\vidly\\startup\\db.js:6:12)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:13:24)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Uncaught Exception dbConnectionLink is not defined","stack":"ReferenceError: dbConnectionLink is not defined\n    at db (F:\\Node Js practise\\vidly\\startup\\db.js:6:12)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:13:24)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47"}
{"level":"error","message":"Uncaught Exception pass is not defined","stack":"ReferenceError: pass is not defined\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\startup\\db.js:3:49)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:13:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)"}
{"level":"error","message":"Uncaught Exception Cannot access 'pass' before initialization","stack":"ReferenceError: Cannot access 'pass' before initialization\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\startup\\db.js:3:49)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:12:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)"}
{"level":"error","message":"Uncaught Exception Cannot access 'pass' before initialization","stack":"ReferenceError: Cannot access 'pass' before initialization\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\startup\\db.js:4:49)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:12:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)"}
{"level":"error","message":"Uncaught Exception Identifier 'config' has already been declared","stack":"F:\\Node Js practise\\vidly\\startup\\config.js:2\nfunction config() {\n^\n\nSyntaxError: Identifier 'config' has already been declared\n    at internalCompileFunction (node:internal/vm:73:18)\n    at wrapSafe (node:internal/modules/cjs/loader:1176:20)\n    at Module._compile (node:internal/modules/cjs/loader:1218:27)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:6:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)"}
{"level":"error","message":"Uncaught Exception Identifier 'config' has already been declared","stack":"F:\\Node Js practise\\vidly\\startup\\config.js:2\nfunction config() {\n^\n\nSyntaxError: Identifier 'config' has already been declared\n    at internalCompileFunction (node:internal/vm:73:18)\n    at wrapSafe (node:internal/modules/cjs/loader:1176:20)\n    at Module._compile (node:internal/modules/cjs/loader:1218:27)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:6:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)"}
{"level":"error","message":"Uncaught Exception Identifier 'config' has already been declared","stack":"F:\\Node Js practise\\vidly\\startup\\config.js:2\nfunction config() {\n^\n\nSyntaxError: Identifier 'config' has already been declared\n    at internalCompileFunction (node:internal/vm:73:18)\n    at wrapSafe (node:internal/modules/cjs/loader:1176:20)\n    at Module._compile (node:internal/modules/cjs/loader:1218:27)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:5:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)"}
{"level":"error","message":"Uncaught Exception Identifier 'config' has already been declared","stack":"F:\\Node Js practise\\vidly\\startup\\config.js:3\nfunction config() {\n^\n\nSyntaxError: Identifier 'config' has already been declared\n    at internalCompileFunction (node:internal/vm:73:18)\n    at wrapSafe (node:internal/modules/cjs/loader:1176:20)\n    at Module._compile (node:internal/modules/cjs/loader:1218:27)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:5:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)"}
{"level":"error","message":"Uncaught Exception Identifier 'config' has already been declared","stack":"F:\\Node Js practise\\vidly\\startup\\config.js:3\nfunction config() {\n^\n\nSyntaxError: Identifier 'config' has already been declared\n    at internalCompileFunction (node:internal/vm:73:18)\n    at wrapSafe (node:internal/modules/cjs/loader:1176:20)\n    at Module._compile (node:internal/modules/cjs/loader:1218:27)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Module.require (node:internal/modules/cjs/loader:1141:19)\n    at require (node:internal/modules/cjs/helpers:110:18)\n    at Object.<anonymous> (F:\\Node Js practise\\vidly\\index.js:5:1)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)"}
{"code":8000,"codeName":"AtlasError","connectionGeneration":0,"level":"error","message":"Unhandled Rejection bad auth : authentication failed","ok":0,"stack":"MongoServerError: bad auth : authentication failed\n    at Connection.onMessage (F:\\Node Js practise\\vidly\\node_modules\\mongodb\\lib\\cmap\\connection.js:202:26)\n    at MessageStream.<anonymous> (F:\\Node Js practise\\vidly\\node_modules\\mongodb\\lib\\cmap\\connection.js:61:60)\n    at MessageStream.emit (node:events:513:28)\n    at processIncomingData (F:\\Node Js practise\\vidly\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:124:16)\n    at MessageStream._write (F:\\Node Js practise\\vidly\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:33:9)\n    at writeOrBuffer (node:internal/streams/writable:392:12)\n    at _write (node:internal/streams/writable:333:10)\n    at Writable.write (node:internal/streams/writable:337:10)\n    at TLSSocket.ondata (node:internal/streams/readable:766:22)\n    at TLSSocket.emit (node:events:513:28)"}
